### DISCUSSION TITLE
合并转发消息显示“消息加载失败，请重新进入”

### DISCUSSION THREAD

**Discussion by VWmin** [mamoe/mirai#1027] [Q&A] [answered]
通过bot发送的合并转发消息（无论是否包含图片）在iPhone端、iPad端无法显示，或者说加载失败？而PC端（TIM）能够显示并加载图片。

发送代码如下：

```JAVA
ForwardMessageBuilder builder = ForwardMessageBuilder(friend);
bulider.says(bot, messageChain);
ForwardMessage message = builder.build();
friend.sendMessage(message);
```

版本：2.4.0

补图
![BC1A528D00FCD333AFDC6AF47D26A22F](https://user-images.githubusercontent.com/34272476/108313192-dedd4700-71f2-11eb-9221-3ddd4157498b.png)
![050100C115433B2050F8C548708EE4C7](https://user-images.githubusercontent.com/34272476/108313199-e43a9180-71f2-11eb-9263-72aa2a786da6.png)
![@WK0M}DMW ({JNAZCEP _6R](https://user-images.githubusercontent.com/34272476/108313204-e7358200-71f2-11eb-8ff2-1010c89e1096.jpg)

**Comment by VWmin** [comment]
刚刚试了下，发送到群的转发消息可以查看，发送给好友的不行

**Comment by VWmin** [comment]
有没有人理我一下

**Comment by Karlatemp** [comment]
github没有convert to issue所以请另外开一个issue

**Comment by Him188** [comment, marked as answer]
我记得好友的合并转发现在就是有点问题的
